torch/onnx/symbolic_opset9.py

```diff
@@ -1378,8 +1378,8 @@ def batch_norm(g, input, weight, bias, running_mean, running_var, training, mome
     out = g.op("BatchNormalization", input, weight, bias, running_mean, running_var,
                epsilon_f=eps,
                momentum_f=1 - momentum,
-               outputs=1 if not sym_help._training_mode else 5)
-    if not sym_help._training_mode:
+               outputs=1 if not training else 5)
```

Comment:
Might need to replicate this for opset 14 BN

Reply:
Well this PR does not have opset 14 updates yet. I'll have to do that after rebase.